compute/etc/sql_exporter/compute_backpressure_throttling_seconds.libsonnet

```diff
@@ -1,6 +1,6 @@
 {
   metric_name: 'compute_backpressure_throttling_seconds',
```

Comment:
```suggestion
  metric_name: 'compute_backpressure_throttling_seconds_total',
```

Reply:
Renamed the file name as well for consistency + flux PR https://github.com/neondatabase/flux-fleet/pull/386